app/src/main/res/drawable/to_progress.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
it was an old "location" icon, I suppose we may delete it now

Reply:
Yes